doc/STARTING_GUIDE.md

```diff
@@ -14,14 +14,25 @@ This step will take a while the first time as it will download all dependencies.
 
 There are three steps necessary to start development, in this case for Android:
 
-1. `make run-clojure` - Compiles Clojure into JavaScript, watches for changes on cljs files, and hot-reloads code in the app
-2. `make run-metro` - Starts metro bundler and watches JavaScript code
-3. `make run-android` - Builds the Android app and starts it on the device
+1. `make run-clojure` - Compiles Clojure into JavaScript, watches for changes on cljs files, and hot-reloads code in the app.
+2. `make run-metro` - Starts metro bundler and watches JavaScript code.
+3. `make run-android` or `make run-ios` - Builds the Android/iOS app and starts it on the device.
 
 The first two will continue watching for changes and keep re-building the app. They need to be ready first.
 The last one will exit once the app is up and ready.
 
-You need to have your emulator or real devices running and visible to adb, before you run `make run-android`.
+## Simulators and Devices
+### Android
+
+You need to have an emulator like [AVD](https://developer.android.com/studio/run/emulator), or [Genymotion](#genymotion-virtualization), or a real device running and visible to `adb`, before you run `make run-android`.
```

Comment:
"..and visible to `adb`",  maybe we can link `adb` here because it might be difficult to what we're referring to

Reply:
Done